packages/react/runtime/__test__/renderToOpcodes.test.jsx

```diff
@@ -380,6 +423,225 @@ describe('renderToOpcodes', () => {
+    expect(rendered[1].type).toStrictEqual(fallbackJsx.type);
+    expect(rendered[2]).toStrictEqual(1);
+  });
+
+  it('should render fallback with suspense 3', async () => {
+    const { Suspender, suspended } = createSuspender();
+
+    const fallbackJsx1 = <text>{`loading1...`}</text>;
+    const fallbackJsx2 = <text>{`loading2...`}</text>;
+
+    const fallbackJsx = (
+      <>
+        {fallbackJsx1}
+        {fallbackJsx2}
+      </>
+    );
+
+    const rendered = renderToString(
+      <Suspense fallback={fallbackJsx}>
+        <view attr={Math.random()}>
+          <Suspender>
+            <text className='foo'>bar</text>
+          </Suspender>
+        </view>
+      </Suspense>,
+    );
+
+    expect(rendered).toMatchInlineSnapshot(`
+      [
+        0,
+        {
+          "children": undefined,
+          "extraProps": undefined,
+          "id": -26,
+          "type": "__Card__:__snapshot_a94a8_test_23",
+          "values": undefined,
+        },
+        3,
+        "loading1...",
+        1,
+        0,
+        {
+          "children": undefined,
+          "extraProps": undefined,
+          "id": -27,
+          "type": "__Card__:__snapshot_a94a8_test_24",
+          "values": undefined,
+        },
+        3,
+        "loading2...",
+        1,
+      ]
+    `);
+
+    expect(rendered[1].type).toStrictEqual(fallbackJsx1.type);
+    expect(rendered[6].type).toStrictEqual(fallbackJsx2.type);
+  });
+
+  it('should render fallback with nested suspense 1', async () => {
+    const { Suspender: Suspender1, suspended: suspended1 } = createSuspender();
+    const { Suspender: Suspender2, suspended: suspended2 } = createSuspender();
+
+    const fallbackJsx1 = <text>loading1...</text>;
+    const fallbackJsx2 = <text>loading2...</text>;
+
+    const rendered = renderToString(
+      <Suspense fallback={fallbackJsx1}>
+        <view attr={Math.random()}>
+          <Suspender1>
+            <text className='foo'>bar</text>
+            <Suspender2 fallback={fallbackJsx2}>
+              <text className='foo'>bar</text>
+            </Suspender2>
+          </Suspender1>
+        </view>
+      </Suspense>,
+    );
+
+    expect(rendered.length).toBe(3);
+    expect(rendered[0]).toStrictEqual(0);
+    expect(rendered[1].type).toStrictEqual(fallbackJsx1.type);
+    expect(rendered[2]).toStrictEqual(1);
+  });
+
+  it('should render fallback with nested suspense 2', async () => {
+    const { Suspender: Suspender1, suspended: suspended1 } = createSuspender();
+    const { Suspender: Suspender2, suspended: suspended2 } = createSuspender();
+    suspended1.resolve();
+    await waitSchedule();
+
+    const fallbackJsx1 = <text>loading1...</text>;
+    const fallbackJsx2 = <text>loading2...</text>;
+    const resolvedJsx1 = <text className='foo'>bar</text>;
+    const resolvedJsx2 = <text className='foo'>bar</text>;
+
+    const rendered = renderToString(
+      <Suspense>
+        <Suspender1 fallback={fallbackJsx1}>
```

Comment:
Do you mean setting fallback on `Suspense`? Or you want a `Suspense` without fallback?

Since it doesn't make any sense to have fallback on Suspender. 

```suggestion
      <Suspense fallback={fallbackJsx1}>
        <Suspender1>
```